services/api/src/models/shop.json

```diff
@@ -0,0 +1,15 @@
+{
+  "attributes": {
```

Comment:
is `attributes` here to keep things open to add metadata on the schema itself later?

Reply:
Yes, I have support for `modelName` right now but I expect lots of controls for example whether the model will be synced to Elasticsearch

Reply:
right makes sense